src/Microsoft.Extensions.Localization.Abstractions/LocalizedString.cs

```diff
@@ -64,6 +77,11 @@ public LocalizedString(string name, string value, bool resourceNotFound)
         public bool ResourceNotFound { get; }
 
         /// <summary>
+        /// The location which was searched for a locatization value.
+        /// </summary>
+        public string LocationSearched { get; }
```

Comment:
I'm still wonder why we have this per `LocalizedString` while the searched location is common for all of them